src/ReactiveUI/Activation.cs

```diff
@@ -54,6 +54,9 @@ public sealed class ViewModelActivator
         /// <value>The deactivated.</value>
         public IObservable<Unit> Deactivated { get { return deactivated; } }
 
+        /// <summary>
```

Comment:
Voted pointless, the return type is `Unit`

![image](https://cloud.githubusercontent.com/assets/127353/22756403/227eedcc-ee9b-11e6-8ccc-92ac6d88480f.png)

See above for another line that can be 🔥